frontend/assets/javascripts/src/modules/giraffe.es6

```diff
@@ -2,6 +2,7 @@ import * as display from 'src/modules/form/validation/display'
 import * as helper from 'src/utils/helper'
 import * as ajax from 'ajax'
 import $ from '$'
+import * as ophan from 'src/modules/analytics/ophan';
```

Comment:
Since you're using ophan.ophan I think you can import {ophan} to pull the ophan property out of the object exported by src/moduls/analytics/ophan